src/extension.ts

```diff
@@ -72,28 +72,36 @@ export function activate(context: vscode.ExtensionContext) {
 
   status.tooltip = 'Path to tag'
 
-  vscode.window.onDidChangeTextEditorSelection(() => {
-    const editor = vscode.window.activeTextEditor
-
-    if (!config.isEnabled || !editor) {
-      return
-    }
-
-    const tagsList = parseTags(editor.document.getText())
-    const position = editor.document.offsetAt(editor.selection.active)
-
-    // Highlight matching tag
-    const match = findMatchingTag(tagsList, position)
-
-    // Tag breadcrumbs
-    if (config.showPath) {
-      updateTagStatusBarItem(status, tagsList, position)
-    }
-
-    if (match && (match.opening !== match.closing || config.highlightSelfClosing)) {
-      tagStyler.decoratePair(match, editor)
-    } else {
-      tagStyler.clearDecorations()
-    }
-  })
+  context.subscriptions.push(
+    vscode.window.onDidChangeTextEditorSelection(() => {
+      const editor = vscode.window.activeTextEditor
+
+      if (!config.isEnabled || !editor) {
+        return
+      }
+
+      const tagsList = parseTags(editor.document.getText())
+      const position = editor.document.offsetAt(editor.selection.active)
+
+      // Highlight matching tag
+      const match = findMatchingTag(tagsList, position)
+
+      // Tag breadcrumbs
+      if (config.showPath) {
+        updateTagStatusBarItem(status, tagsList, position)
+      }
+
+      if (match && (match.opening !== match.closing || config.highlightSelfClosing)) {
+        tagStyler.decoratePair(match, editor)
+      } else {
+        tagStyler.clearDecorations()
+      }
+    })
+  )
+
+  context.subscriptions.push(
+    vscode.commands.registerCommand('highlight-matching-tag.jumpToMatchingTag', async () => {
```

Comment:
I think this is a little cleaner:

`vscode.commands.registerCommand('highlight-matching-tag.jumpToMatchingTag', jumpToMatchingTag)`